bentoml/sklearn.py

```diff
@@ -130,38 +140,37 @@ def __init__(
         self._model_store = model_store
         self._model_info = model_info
         self._model_file = model_file
-        self._parallel_ctx = parallel_backend(
-            "threading", n_jobs=self.num_concurrency_per_replica
-        )
+        self._backend = "loky"
         self._function_name = function_name
 
     @property
     def num_concurrency_per_replica(self) -> int:
-        # NOTE: sklearn doesn't use GPU, so return max. no. of CPU's.
-        return int(round(self.resource_quota.cpu))
+        return 1
 
     @property
     def num_replica(self) -> int:
-        # NOTE: sklearn doesn't use GPU, so just return 1.
-        return 1
+        return int(round(self.resource_quota.cpu))
 
     @property
     def required_models(self) -> t.List[Tag]:
         return [self._model_info.tag]
 
-    # pylint: disable=arguments-differ,attribute-defined-outside-init
-    def _setup(self) -> None:  # type: ignore[override]
+    # pylint: disable=attribute-defined-outside-init
+    def _setup(self) -> None:
         self._model = joblib.load(filename=self._model_file)
+        self._infer_func = getattr(self._model, self._function_name)
 
     # pylint: disable=arguments-differ
-    def _run_batch(  # type: ignore[override]
+    def _run_batch(
         self,
-        *args: t.Union[np.ndarray, "pd.DataFrame"],
-        **kwargs: t.Union[np.ndarray, "pd.DataFrame"],
-    ) -> "np.ndarray":
-        func = getattr(self._model, self._function_name)
-        with self._parallel_ctx:
-            return func(*args, **kwargs)
+        *args: t.Union["np.ndarray[t.Any, np.dtype[t.Any]]", "DataFrame"],
```

Comment:
I think it's the same case for sklearn, this should be just:
```
def _run_batch(self, batch_input: t.Union["np.ndarray[t.Any, np.dtype[t.Any]]", "DataFrame"])
```

Reply:
noted